scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+# A postprocessing workflow for GATK CNV.
+#
+# THIS CANNOT BE RUN IN TUMOR-ONLY MODE.  A MATCHED NORMAL IS REQUIRED.
+#
+# Internally, we have shown that this workflow improves sensitivity and precision.  Additionally, this workflow is quite
+#  cheap to run.  Note that auxiliary files are required and without these exact files, performance may suffer.
+#
+# UNSUPPORTED and we may modify this WDL before we fully support it.
+#
+# This will also generate absolute (https://software.broadinstitute.org/cancer/cga/absolute) compatible files.
+#  Currently, the balanced-segment calling is extremely ham-fisted.
```

Comment:
"ham-fisted" is a bit weird and not confidence inducing...why not just say it is extremely basic or heuristic?

Reply:
Actually, I see below that it is back to being based on a hard threshold, so I'd just go ahead and say that here.

Reply:
Done